src/Security/MemberAuthenticator/ChangePasswordHandler.php

```diff
@@ -163,7 +163,7 @@ protected function createChangePasswordResponse(): array|HTTPResponse
         $session = $this->getRequest()->getSession();
         $hash = $session->get('AutoLoginHash');
         if ($hash) {
-            if (!Member::get()->filter(['AutoLoginHash' => $hash])) {
+            if (!Member::get()->filter(['AutoLoginHash' => $hash])->exists()) {
```

Comment:
I could have changed `filter()` to `find()` but using `exists()` is more efficient both in the db itself (uses a `SELECT 1 CASE EXISTS (SELECT <where etc>) ELSE 0` query) and in PHP-land since it's a boolean and therefore we don't have to hydrate a whole `DataObject` record just to throw it away.